.editorconfig

```diff
@@ -4,13 +4,11 @@
 ###############################
 # All files
 [*]
+charset = utf-8
 indent_style = space
+indent_size = 4
```

Comment:
We've got a few `yml`, `xml`, and `json` files that are normally `indent_size = 2`. I think this will affect those.

Reply:
Good catch, fixed.